packages/binaryninja-free.vm/binaryninja-free.vm.nuspec

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>binaryninja-free.vm</id>
```

As documented in the [_id_ section in our Wiki](https://github.com/mandiant/VM-Packages/wiki/Chocolatey-packages#id):
> The `id` is the tool name (usually the file name with the `.exe`) lowercased and with the following format: `toolname.vm` or `tool-name.vm`. The `id` must be identical to the tool directory name.

```suggestion
    <id>binaryninja.vm</id>
```